lib/fastlane/plugin/revenuecat_internal/actions/bump_phc_version_create_pr_action.rb

```diff
@@ -0,0 +1,109 @@
+        end
+
+        Helper::RevenuecatInternalHelper.replace_version_number(version_number,
+                                                                new_version_number,
+                                                                files_to_update,
+                                                                [])
+
+        return unless open_pr
+
+        Helper::RevenuecatInternalHelper.commit_changes_and_push_current_branch("Version bump for #{new_version_number}")
+
+        pr_title = "Updates purchases-hybrid-common to #{new_version_number}"
+        label = 'dependencies'
+        body = pr_title
+
+        if automatic_release
+          body = "**This is an automatic release.**\n\n#{body}"
+          pr_title = "[AUTOMATIC] #{pr_title}"
+        end
+
+        Helper::RevenuecatInternalHelper.create_pr_to_main(pr_title, body, repo_name, new_branch_name, github_pr_token, [label])
+      end
+
+      def self.description
+        "Bumps purchases-hybrid-common dependency version and creates PR with changes."
+      end
+
+      def self.authors
+        ["César de la Vega"]
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :current_version,
+                                       description: "Current version of the SDK",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :files_to_update,
+                                       env_name: "RC_INTERNAL_FILES_TO_UPDATE_VERSION",
+                                       description: "Files that contain the version number and need to have it updated",
+                                       optional: false,
+                                       type: Array),
+          FastlaneCore::ConfigItem.new(key: :repo_name,
+                                       env_name: "RC_INTERNAL_REPO_NAME",
+                                       description: "Name of the repo of the SDK",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :github_pr_token,
+                                       env_name: "GITHUB_PULL_REQUEST_API_TOKEN",
+                                       description: "Github token to use to create the release PR",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :next_version,
+                                       description: "Next version of the SDK",
+                                       optional: true,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :automatic_release,
+                                       description: "If this is an automatic release",
+                                       optional: true,
```

Comment:
Since this and `open_pr` are optional, should we add a default value?

Reply:
I think I am going to set it to false, and rename the action as you suggested in the other comment